tests/validation/config.json

```diff
@@ -818,6 +818,11 @@
             "title": "Visibility",
             "playgroundId": "#PXC9CF#2",
             "referenceImage": "visibility.png"
+        },
+        {
+            "title": "GLTF Extension EXT_mesh_gpu_instancing",
+            "playgroundId": "#QFIGLW#3",
```

Comment:
Most of the test assets are located in the [meshes library](https://github.com/BabylonJS/MeshesLibrary/tree/master/Tests) to avoid bloating the Babylon.js repo. It probably makes sense to do that for this too?

Reply:
yes, makes sense. I'll add this asset to the meshes library.